Integrations/Anomaliv2/Anomaliv2.py

```diff
@@ -0,0 +1,882 @@
+
+    if model == 'signature':
+        description = description.get('notes', None)
+    elif model == 'tipreport':
+        description = description.get('body', None)
+    else:
+        description = description.get('description', None)
+
+    if not description:
+        demisto.results(F"No description found for Threat Model {model.title()} with id {id}")
+        sys.exit()
+
+    demisto.results(fileResult(F"{model}_{id}.html", description.encode(encoding='UTF-8')))
+
+
+def get_iocs_by_model(model, id, limit="20"):
+    """
+        Returns list of indicators associated with specific Threat Model by model id.
+    """
+    params = build_params(limit=limit)
+    iocs_list = http_request("GET", F"v1/{model}/{id}/intelligence/", params=params).get('objects', None)
+
+    if not iocs_list:
+        demisto.results(F"No indicators found for Threat Model {model.title()} with id {id}")
+        sys.exit()
+
+    iocs_context = parse_indicators_list(iocs_list)
+
+    ec = {
+        'ThreatStream.Model(val.ModelID == obj.ModelID && val.ModelType == obj.ModelType)': {'ModelType': model.title(),
+                                                                                             'ModelID': id,
+                                                                                             'Indicators': iocs_context},
+    }
+    human_readable = tableToMarkdown(F"Indicators list for Threat Model {model.title()} with id {id}", iocs_context)
+
+    return_outputs(human_readable, ec, iocs_list)
+
+
+def create_model(model, name, is_public="false", tlp="red", tags=None, intelligence=None, description=None):
+    """
+        Creates Threat Model with basic parameters.
+    """
+    data = {k: v for k, v in locals().items() if v and k not in ['model', 'tags', 'intelligence']}
+    if tags:
+        data['tags'] = [t.strip() for t in tags.split(',')]
+    if intelligence:
+        data['intelligence'] = [i.strip() for i in intelligence.split(',')]
+
+    created_id = http_request("POST", F"v1/{model}/", data=json.dumps(data), params=CREDENTIALS).get('id', None)
+
+    if created_id:
+        ec = {'ThreatStream.CreatedModel': {'ID': created_id, 'Model': model.title()}}
+        return_outputs(f"{model.title()} Threat Model was created with {created_id} id", ec, created_id)
+    else:
+        demisto.results(f"{model.title()} Threat Model was not created. Check the input parameters")
+
+
+def update_model(model, model_id, name=None, is_public="false", tlp=None, tags=None, intelligence=None,
+                 description=None):
+    """
+        Updates a ThreatStream model with parameters. In case one or more optional parameters are
+        defined, the previous data is overridden.
+    """
+    data = {k: v for k, v in locals().items() if v and k not in ['model', 'model_id', 'tags', 'intelligence']}
+    if tags:
+        data['tags'] = [t.strip() for t in tags.split(',')]
```

Comment:
set tags with `is_array` flag. also, consider the option you get a list here and not a list.